www/ssc/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   cmake 1.1
+revision                    0
+
+github.setup                devongarde ssc 0.1.56 v
+github.tarball_from         releases
+categories                  www
+platforms                   {darwin >= 18} {openbsd >= 7.2}
+supported_archs             x86_64 arm64
+license                     GPL-3
+maintainers                 { @devongarde } openmaintainer
+description                 an opinionated static website nitpicker
+long_description            ${name}, the Static Site Checker, is an opinionated nitpicker for static \
+                            websites, analysing HTML, SVG, MathML, and more, including ontologies. \
+                            It can produce detailed reports, 'repaired' HTML with resolved server \
+                            side includes, site statistics, etc.. Consider it, at best, alpha.
+
+distname                    ${name}-v${version}
+distfiles                   ssc-v${version}.tgz
+extract.suffix              .tgz
+extract.post_args           "| tar -xzf -"
+
+homepage                    https://ssc.lu/
+master_sites                https://github.com/devongarde/ssc/releases/download/v${version}
+
+checksums                   rmd160  df2784a8d7671568cc3bd87b8a7f0ba9b0d9c809 \
+                            sha256  1ee28fec98d7910f87f7120cbaa517837259f4eced34491576c2b8e2834b2ce3 \
+                            size    23016387
+
+depends_lib                 port:boost \
```

Comment:
Don't add a `port:boost` dependency  manually. Instead, include the boost portgroup and make use of its facilities. Also, don't rely on its default boost version number. Use the command that that portgroup defines for specifying the boost version, and choose the newest version that this software is compatible with (and that exists in MacPorts), and update that as needed in the future.